### PR TITLE
ci: init

### PR DESCRIPTION
**Pull request by nugaon**
Build and publish docker images by Github CI

### CODE REVIEW

File: scripts/publish-environment.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+MY_PATH=$(dirname "$0")
+MY_PATH=$( cd "$MY_PATH" && pwd )
+
+DOCKER_IMAGES=$(docker image ls -qaf reference="$BEE_IMAGE_PREFIX/$BEE_ENV_PREFIX*:$BEE_VERSION")
+for DOCKER_IMAGE in $DOCKER_IMAGES
+do
+  echo "$DOCKER_IMAGE"
+  docker push "$DOCKER_IMAGE"
```
Comment (AuHau) [inline scripts/publish-environment.sh:9]: Hmm, if I understand the images will be pushed with a tag similar to `$BEE_IMAGE_PREFIX/$BEE_ENV_PREFIX*:$BEE_VERSION` right?

I am not sure if this tag scheme is sufficient for our use case as I can imagine we might have multiple versions for given `BEE_VERSION`, which is then question if we pronounce these tags as "immutable" or "mutable". If mutable then there is no issue, but honestly, I am not a big fan of that as it is hard to reason what version are you using.

**Review by AuHau — COMMENTED** [review]
Few suggestions. One thing I am not sure about is that these published images are if I am not mistaken "blank". They are funded and so on, but there is still no data/traffic generated. I think that is the end goal right? So I assume later on there will be some script that will generate the content of the nodes as we need right? But I guess that is not the scope of this PR?

**Comment by nugaon** [comment]
> Few suggestions. One thing I am not sure about is that these published images are if I am not mistaken "blank". They are funded and so on, but there is still no data/traffic generated. I think that is the end goal right? So I assume later on there will be some script that will generate the content of the nodes as we need right? But I guess that is not the scope of this PR?

Yes, you are right, these images are black, only features they have their keys are known and its overlay addresses have been funded. 
As we discussed on one of our roundtable, we are going to generate this data traffic on `bee-js` side as @agazso  proposed.

File: scripts/build-environment.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+MY_PATH=$(dirname "$0")
+MY_PATH=$( cd "$MY_PATH" && pwd )
```
[the remark this answers is not in the document]
Comment (nugaon) [reply scripts/build-environment.sh:3]: I'm not a bash expert, but I couldn't modify or event create global variables from these scripts.

File: scripts/publish-environment.sh (hunk 1 above)
Comment (nugaon) [reply scripts/publish-environment.sh:9]: you got my concerns, I think the docker pushes can act in a mutable way, but I'm not sure about the pulls...

Comment (nugaon) [reply scripts/publish-environment.sh:9]: as I notice on the CI jobs here, it always pulls the docker images so it shouldn't cause problem on CI level

**Pull request merged**